docs/core/diagnostics/sample-debug-target.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Sample debug target
+description: Introduction to the sample debug target used in the scenarios.
+author: sdmaclea
+ms.author: stmaclea
+ms.date: 10/14/2019
+---
+# Sample debug target
+
+The scenarios are implemented using a `webapi` target with methods that trigger undesirable behaviors for us to diagnose.
+
+## Sample source
+
+The sample code is in [DiagnosticScenarios](https://github.com/dotnet/samples/blob/master/core/diagnostics/DiagnosticScenarios). The sample is a basic `webapi` with a single added file [DiagnosticScenarios.cs](https://github.com/dotnet/samples/blob/master/core/diagnostics/DiagnosticScenarios/Controllers/DiagnosticScenarios.cs).
+
+The simplest way to acquire to source is to clone the [samples repo](https://github.com/dotnet/sample)
```

Comment:
It might be better to create a release or a zip file with the contents of the tool for this so people can download this more easily?

Reply:
I chose to ignore this for now.  As a developer the git clone doesn't bother me at all. Unzip is just as hard.

I can see it might be a bit clumsy...

Reply:
https://github.com/dotnet/samples/releases/download/82507/Diagnostic_Scenarios.zip

Reply:
> 
> 
> I chose to ignore this for now. As a developer the git clone doesn't bother me at all. Unzip is just as hard.
> 
> I can see it might be a bit clumsy...

I agree with @mairaw, for people with slow internet connection, it's much faster to download a zip file for the content I want, rather than cloning a whole big repo like the dotnet/samples.

Reply:
I removed the sample-debug-target.md from this PR.  The PR now uses the readme in the samples repo.  I will update the content there.